modules/oadp-kubevirt-setting-resource-limits-and-requests.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-kubevirt-setting-resource-limits-and-requests_{context}"]
+= Velero CPU and memory resource allocations for {VirtProductName}
+
+You set the CPU and memory resource allocations for the `Velero` pod by editing the  `DataProtectionApplication` custom resource (CR) manifest.
+
+.Prerequisites
+
+* You must have the OpenShift API for Data Protection (OADP) Operator installed.
+
+.Procedure
+
+* Edit the values in the `spec.configuration.velero.podConfig.ResourceAllocations` block of the `DataProtectionApplication` CR manifest, as in the following example:
++
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: <dpa_sample>
+spec:
+...
+  configuration:
+    velero:
+      podConfig:
+        nodeSelector: <node selector> <1>
```

Comment:
```suggestion
        nodeSelector: <node_selector> <1>
```

Reply:
If this is a user-replaced value, use the following syntax: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values